.devcontainer/codespaces/devcontainer.json

```diff
@@ -23,6 +23,8 @@
     }
   },
 
+  "remoteUser": "root",
```

Comment:
This aligns with the non-codespace one, but is also sort of a stab at resolving a codespace deploy issue (from main just now):

```
2024-06-06 22:31:05.276Z: There was an error while trying to write to
`/usr/local/rvm/gems/default/cache/bundler/git`. It is likely that you need to
grant write permissions for that path.
```